data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticle.java

```diff
@@ -176,8 +175,8 @@ public void start(final Promise<Void> startPromise) {
     // Set up the secret watcher
     private void setupSecretWatcher() {
         try {
-            File file = new File(secretVolumePath + "/tls.crt");
-            this.secretWatcher = new FileWatcher(file, this::updateServerConfig);
+            new Thread(this.secretWatcher).start();
```

Comment:
No need to start a new thread here. If you look into the source code for FileWatcher, a new thread will be started there.